src/utilcode/pedecoder.cpp

```diff
@@ -1406,21 +1406,20 @@ CHECK PEDecoder::CheckILOnly() const
 
     CHECK(CheckCorHeader());
 
-
     // Allow only verifiable directories.
```

Comment:
Would it be better to just have an early out if the image is R2R?

Reply:
Are all the checks in this function useless for R2R? I just don't know. They look like they have a purposes

Reply:
All these checks mattered for partial trust. They are just nice-to-have sanity checks now.

Reply:
>All these checks mattered for partial trust

Sounds good. Thanks for letting me know. I'll make it an early out for R2R then